cli/cli.go

```diff
@@ -792,6 +815,12 @@ func confirmInstallDependencies(p ui.Prompter, o *CliCmdOptions) (err error) {
 }
 
 func confirmEnableMEVBoost(p ui.Prompter, o *CliCmdOptions) (err error) {
+	if o.nodeSetup == LidoNode {
```

Comment:
Same. Please try to remove these Ifs as much as possible and replace them with something more functional. A `nodeSetup` could be a complete interface with several methods that wraps custom logic such as the one contained within the conditional. Visualize how the code would look like if we have 5 different `nodeSetup`. 

This refactor usually only makes sense after applying the [rule of three](https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming)), which will be true as we will support Starknet soon as a different `nodeSetup`

Reply:
[Done](https://github.com/NethermindEth/sedge/pull/389/commits/381a99f8d4edc125d9a267655d844c558d02a21d)